.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,11 +4,13 @@ Issue:
 
 ## How to test
 
-Is this testable with Jest or Chromatic screenshots?
-Does this need a new example in the kitchen sink apps?
-Does this need an update to the documentation?
+- Is this testable with Jest or Chromatic screenshots?
+- Does this need a new example in the kitchen sink apps?
+- Does this need an update to the documentation?
 
 If your answer is yes to any of these, please make sure to include it in your PR.
 
-For maintainers only: Please tag your pull request with at least one of the following:
+**For everybody:** Please submit all PRs to the `next` branch unless they are specific to current release. Storybook maintainers cherry-pick bug and documentation fixes into the `master` branch as part of the release process, so you shouldn't need to worry about this.
```

Comment:
let's include both notes in `<!--`/`-->` to hide them from actual PRs

Reply:
See e.g https://raw.githubusercontent.com/facebook/create-react-app/master/.github/ISSUE_TEMPLATE.md